### ISSUE TITLE
maf2vcf can't handle big MAFs on CentOS6

### ISSUE THREAD

**Issue by ckandoth** [mskcc/vcf2maf#79]
Tried to annotate a MAF with 255001 variants, as follows:
```bash
perl /opt/common/CentOS_6-dev/vcf2maf/v1.6.9/maf2maf.pl --input-maf icgc.release_22.maf  --output-maf icgc.release_22.vep.maf 2> stderr.log
```
Which returns the following error, followed by warnings about uninitialized `$ref_from_fasta`:
```bash
Can't exec "/opt/common/CentOS_6-dev/bin/current/samtools": Argument list too long at /opt/common/CentOS_6-dev/vcf2maf/v1.6.9/maf2vcf.pl line 99.
```
@alexpenson - Thanks for reporting this.

**Comment by ckandoth** [comment]
This is now fixed in 1cd58f6 where `samtools faidx` is run on chunks of 100k loci, instead of everything in the MAF at once. @alexpenson can you test `cmo_maf2maf --version develop` on luna?

**Comment by ckandoth** [comment]
I went ahead and tested this and it worked OK. I'm going to replace the v1.6.9 tag with commit 1cd58f6, even though that's not a nice thing to do. :-| Won't affect us internally at MSK, but I hope this message gets to the external users who downloaded v1.6.9 earlier.
